### ISSUE TITLE
[Supporting the platform-MCIO DCX-O1NLD Land]: In High contrast (Aquatic, Desert, Dusk, Night sky) themes, focus indicator is not visible on the table header and rows.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7252]
Description:
Color Sensitive users who rely on High contrast modes would be impacted if the context menu icon is not visible to them. Color-sensitive users may not access the control as they cannot see the control in the High contrast theme.

Test Environment:
OS Build: Version 21H2(22000.856)
Browser: Microsoft Edge Version 105.0.1343.33 (Official build) (64-bit)
Screen reader: Non-Visual Desktop Access (NVDA) (2022.2)

Pre-Requisites:
Turn on System High contrast themes. In system settings->Contrast themes.
 
Repro Steps:
1. Open [Angular Grid | Excel Style Filtering | Infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)
2. Angular excel style filtering page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the focus is visible in high contrast themes or not.

Actual Result:
In High contrast (Aquatic, Desert, Dusk, Night sky) themes, the focus indicator is not visible on the table header and rows present on the 'Excel Filtering in Angular Grid' page.

Expected Result:
In High contrast (Aquatic, Desert, Dusk, Night sky) themes, the focus indicator should be visible on the table header and rows present on the 'Excel Filtering in Angular Grid' page.

Additional Information:
Same issue Reproes throughout the application.

Attachment:
[Proof 1]<img width="301" alt="Picture1" src="https://user-images.githubusercontent.com/93735775/189976632-9191bf0b-dc47-4938-a38a-531afa16504a.png">

[Proof 2]<img width="452" alt="Picture2" src="https://user-images.githubusercontent.com/93735775/189976679-eb92bea6-1407-4f3e-a9bd-38bc07ff5b4f.png">

[Proof 3]<img width="301" alt="Picture3" src="https://user-images.githubusercontent.com/93735775/189976725-41a5d768-375b-4d70-813a-a1887d62ae3a.png">

[Proof 4]<img width="301" alt="Picture4" src="https://user-images.githubusercontent.com/93735775/189976756-08dc317b-94b1-4296-b395-f21443b9d9fd.png">

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
